astropy/coordinates/earth.py

```diff
@@ -10,11 +10,13 @@
 
 import numpy as np
 from .. import units as u
+from .. import constants as consts
 from ..units.quantity import QuantityInfoBase
 from ..utils.exceptions import AstropyUserWarning
 from ..utils.compat.numpycompat import NUMPY_LT_1_12
 from .angles import Longitude, Latitude
-from .representation import CartesianRepresentation
+from .matrix_utilities import matrix_transpose
```

Comment:
Should have seen this when I made the PR to your PR, but we don't need this any more.

Reply:
yep, good point